docs/content/en/docs/concepts/performance-stats.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "BPF programs statistics"
+weight: 5
+description: "Monitor bpf programs statistics"
+---
+
+This page shows you how to monitor bpf programs statistics.
+
+## Concept
+
+The BPF subsystem provides performance data for each loaded program and tetragon
+exports that in metrics or display that in terminal in top like tool.
+
+## In terminal
+
+The tetra command allows to display loaded BPF programs in terminal with:
+
+```shell
+# tetra debug progs
```

Comment:
```suggestion
tetra debug progs
```
please remove `#` or `$`. I know it's a debatable choice, but we decided in the doc to make it copy-pastable everywhere else. We follow this https://kubernetes.io/docs/contribute/style/style-guide/#don-t-include-the-command-prompt